functions.py

```diff
@@ -10,6 +10,10 @@ def draw_layout(game):
   start_position = (game.rect.x, game.play_area_height)
   end_position = (game.rect.right - game.border_width, game.play_area_height)
   pygame.draw.line(game.screen, game.border_colour, start_position, end_position, game.border_width)
+  display_text = game.font.render("Score:", True, game.text_colour)
+  display_text_rect = display_text.get_rect()
+  display_text_rect.center = pygame.Rect(40, 740, 80, 40).center
```

Comment:
This text and its rect are being recreated on every call with no change.

Reply:
I call this functions only once so is it a problem then. If it is then how could I solve this?